meta-xt-domu/recipes-kernel/linux/linux-virtio-armv8/0002-DEBUG-Print-retrieved-stream-ID-backend_domid-for-bo.patch

```diff
@@ -0,0 +1,44 @@
+From 8b43057c9c4edb2a45165c14c1ea3fed4efd4942 Mon Sep 17 00:00:00 2001
+From: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
+Date: Sun, 23 Oct 2022 19:57:46 +0300
+Subject: [PATCH 2/2] [DEBUG] Print retrieved stream ID (backend_domid) for
```

Comment:
Do we really need this patch? Looks like it is something for debugging.

Reply:
Okay then. Could you please mention this in the commit message? I.e. that this new machine is for virtio, and that new kernel recipe has debugging kernel patches, that we need because...

This will save as a lot of headache for anyone who will try to understand what is going on.

Reply:
Done